src/qlik-on-demand-reporting.js

```diff
@@ -74,49 +74,29 @@ function(
         format = options.format,
         selections = getSelectionByApi();
 
-        return selections.then(function(allFieldSelections) {				
-            return hlp.doGetConnections(conn.server, conn.app).then(function(response) {
-                var connectionId;
```

Comment:
Need to fetch a valid connection id here if none is set. Otherwise this will fail for all current users after they've updated.